### PR TITLE
note about using the same remote as ext cache and for push/pull

### LINKED ISSUE #516: external outputs: add a note about using the same remote as ext cache and for push/pull
Discord context https://discordapp.com/channels/485586884165107732/563406153334128681/606197772672172044

### PR DESCRIPTION
**Pull request by ryokugyu**
Fix #516

### CODE REVIEW

File: static/docs/commands-reference/remote_default.md [hunk 1]
```diff
@@ -90,6 +90,10 @@ Change default remote value:
 $ dvc remote default mynewremote
 ```
 
+> Avoid using the same remote that you are using for `dvc push`, `dvc pull`,
```
Comment (shcheklein) [inline static/docs/commands-reference/remote_default.md]: This is not the right place for this note. It should to the external outputs doc. Also, it should be a little bit more informative.

Comment (ryokugyu) [reply static/docs/commands-reference/remote_default.md]: need to remove this note from here? and just kept it in the external output section only? @shcheklein

Comment (shcheklein) [reply static/docs/commands-reference/remote_default.md]: yes, I think it's enough to keep that information and notes in the external outputs section

**Comment by ryokugyu** [comment]
added extra information about the same, and removed the note from the `default remote` section.

File: static/docs/commands-reference/cache_dir.md [hunk 2]
```diff
@@ -65,4 +65,10 @@ $ cat .dvc/config
 ...
 ```
 
+> Avoid using the same remote that you are using for `dvc push`, `dvc pull`,
```
Comment (shcheklein) [inline static/docs/commands-reference/cache_dir.md]: not need to put this note here, let's not complicate it

File: static/docs/user-guide/external-outputs.md [hunk 3]
```diff
@@ -19,6 +19,12 @@ an [external cache](/doc/commands-reference/config#cache) location that will be
 used by dvc to store versions of your external file. Non-cached external outputs
 (specified using `-O`) do not require external cache to be setup.
 
+> Avoid using the same remote that you are using for `dvc push`, `dvc pull`,
+> `dvc fetch` as external cache for your external outputs, because it may cause
+> possible checksum overlaps. `etags` on remote storage might theoretically
```
Comment (shcheklein) [inline static/docs/user-guide/external-outputs.md]: it's not always `etag` it's specific to remote storage. You can say just. `Checksum for some data file on an external storage can potentially collide with checksum generated locally for a different file, with a different content.`

Comment (ryokugyu) [reply static/docs/user-guide/external-outputs.md]: updated

File: static/docs/commands-reference/cache_dir.md (hunk 2 above)
Comment (ryokugyu) [reply static/docs/commands-reference/cache_dir.md]: removed

File: static/docs/user-guide/external-outputs.md [hunk 4]
```diff
@@ -19,6 +19,12 @@ an [external cache](/doc/commands-reference/config#cache) location that will be
 used by dvc to store versions of your external file. Non-cached external outputs
 (specified using `-O`) do not require external cache to be setup.
 
+> Avoid using the same remote that you are using for `dvc push`, `dvc pull`,
```
Comment (shcheklein) [inline static/docs/user-guide/external-outputs.md]: let's change `remote` -> `remote location`

Comment (ryokugyu) [reply static/docs/user-guide/external-outputs.md]: done

**Comment by shcheklein** [comment]
thanks 🎉

**Pull request merged**